docs/architecture.md

```diff
@@ -0,0 +1,34 @@
+# Architecture
+***
+
+While we prefer in-code and live documentation over wiki pages, these are best suited to single services and we don't have a place to document broader processes that involve the interaction of several services. This section aims to provide that overview of how various services fit together.
+
+## Supporter data flow
+
+```mermaid
+graph LR
+  fundrasing-signups["Fundraising Sign-ups (Tom)"] --- event
```

Comment:
Most minor thing of all time but there's a missing `i` here and also, would be nice to stick to the same naming conventions - I've been going with 'Fundraiser Signups'

Reply:
Thanks, well spotted! Agree that names should be consistent everywhere.